allure-testng-adaptor/src/main/java/ru/yandex/qatools/allure/testng/AllureTestListener.java

```diff
@@ -2,13 +2,13 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.testng.IConfigurationListener;
+import org.testng.ISuite;
+import org.testng.ISuiteListener;
 import org.testng.ITestContext;
-import org.testng.ITestListener;
 import org.testng.ITestNGMethod;
 import org.testng.ITestResult;
-import org.testng.SkipException;
-import org.testng.TestException;
+import org.testng.internal.IResultListener;
```

Comment:
This is just simple interface aggregation class  which  combines IConfigurationListener and ITestListener interfaces.There are a lot of useful public classes in the Internal package so I don't think that they will be changed soon. But if you would like to be on the safe side I can replace this interface with other interfaces which are not in the internal package